lib/smart_app_launch/backend_services_group.rb

```diff
@@ -0,0 +1,237 @@
+require_relative 'authorization_request_builder'
+
+module SMARTAppLaunch
+  class SMARTBackendServices < Inferno::TestGroup
+    title 'Backend Services'
+    short_description 'Demonstrate SMART Backend Services Authorization'
+
+    id :smart_backend_services
+
+    input :bulk_token_endpoint,
```

Comment:
The input ids should be updated to reflect that these are backend services inputs not bulk data inputs.

Reply:
All inputs were updated.